app/src/main/java/org/literacyapp/synchronization/GroupDeleteHelper.java

```diff
@@ -0,0 +1,95 @@
+package org.literacyapp.synchronization;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.Collection;
+import java.util.Iterator;
+
+import android.content.Context;
+import android.net.wifi.p2p.WifiP2pGroup;
+import android.net.wifi.p2p.WifiP2pManager;
+import android.util.Log;
+import android.net.wifi.p2p.WifiP2pManager.Channel;
+
+public class GroupDeleteHelper   {
```

Comment:
@elich11 What does "group" mean in this context? Can you add some JavaDoc explaining how groups work?